server/src/main/java/org/elasticsearch/script/SortedSetDocValuesStringFieldScript.java

```diff
@@ -23,7 +24,7 @@ public class SortedSetDocValuesStringFieldScript extends StringFieldScript {
     boolean hasValue = false;
 
     public SortedSetDocValuesStringFieldScript(String fieldName, SearchLookup searchLookup, LeafReaderContext ctx) {
-        super(fieldName, Map.of(), searchLookup, ctx);
+        super(fieldName, Map.of(), searchLookup, ErrorBehaviour.FAIL, ctx);
```

Comment:
good choice. This is a case where we reused the existing script based doc_value infrastructure for doc_value only fields. There is effectively no script here, hence no need to configure on error behaviour. Same for the one above.